chain/network-primitives/src/network_protocol/mod.rs

```diff
@@ -223,6 +223,18 @@ pub enum RoutedMessageBody {
     PartialEncodedChunkForward(PartialEncodedChunkForwardMsg),
 }
 
+impl RoutedMessageBody {
+    // Return whether this message is important.
+    // In routing logics, we send important messages multiple times to minimize the risk that they are
+    // lost
+    pub fn is_important(&self) -> bool {
+        match self {
+            RoutedMessageBody::BlockApproval(_) | RoutedMessageBody::PartialEncodedChunk(_) => true,
```

Comment:
add a comment why these are important (especially to explain the PartialEncodedChunk one)